src/lib/components/Authenticate/Login.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+  import { createEventDispatcher } from 'svelte';
+  const dispatch = createEventDispatcher();
+</script>
+
+<div>
+  <form on:submit={() => console.log('Submit')}>
```

Comment:
This will be taken care of in a next PR using https://github.com/whizzbit/nexus-api/commit/372d062c78a889c9f2fb4b5a5dfdbc0fbc751f72